whistl/datasets.py

```diff
@@ -256,3 +440,65 @@ def __getitem__(self, idx):
     def __len__(self):
         '''Provides the number of samples in the dataset'''
         return len(self.labels)
+
+
+class CompendiumDataset(Dataset):
```

Comment:
This class isn't being used anywhere yet, correct? I'm assuming that's coming in a future PR?

Reply:
Correct, next PR should be the start of actually training models on compendium data